src/Codec/Compression/LZ4/Conduit.hsc

```diff
@@ -215,7 +224,7 @@ withScopedLz4fCompressionContext f =
 lz4fCreateCompressonContext :: (HasCallStack) => IO Lz4FrameCompressionContext
 lz4fCreateCompressonContext = do
   ctxForeignPtr :: ForeignPtr (Ptr LZ4F_cctx) <- mallocForeignPtr
-  -- Note [Initialize LZ4 context pointer to NULL]:
```

Comment:
The backets are intentional: I use this as "tags" to label comment sections explicitly, and other places link to this, e.g. `See note [Initialize LZ4 context pointer to NULL]:` in another place in this file.

That is so that I can explain something once, and have the other places that rely on that invariant link to that, without repeating myself. With the brackets it's clearer that that is the label that you need to search for.